### PR TITLE
307 (scene expert individual page)

### LINKED ISSUE #307: As a user I'd like to browse the information for each Scene Builder in their individual pages so i can learn more about them
When a user clicks on a card in the Scene Builders landing page they should be redirected to a dedicated page for the Scene Builder. The design of the page is found [here](https://www.figma.com/file/gMFxv9DjmUkHbZQSe4lu5E/EnterDAO-Projects?node-id=7509%3A16038).

The page should include the following components:

1. Breadcrumb - same as in the property details page (element)
2. Cover photo
3. Profile photo
4. Scene Builder Name
5. Type of the Scene Builder
6. Definition
7. Icons for LinkedIn / Twitter / Discord
8. Long Description
9. Website
10. Location
11. Languages
12. Tags
13. Price point (blue dollar icons represent value)
14. Send a message button (element)

### PR DESCRIPTION
**Pull request by zetske**
**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #307 

**Special notes for your reviewer**:

**Checklist**
- [x] Documentation added
- [x] Breadcrumb - same as in the property details page (element)
- [x] Cover photo
- [x] Profile photo
- [x] Scene Builder Name
- [x] Type of the Scene Builder
- [x] Definition
- [x] Icons for LinkedIn / Twitter / Discord
- [x] Long Description
- [x] Website
- [x] Location
- [x] Languages
- [x] Tags
- [x] Price point (blue dollar icons represent value)
- [x] Send a message button (element)

### CODE REVIEW

File: src/modules/land-works/views/scene-expert-view/index.tsx [hunk 1]
```diff
@@ -27,8 +27,10 @@ const SceneExpertView: FC = () => {
   useEffect(() => {
     (async () => {
       const sceneProv = await getSceneProviders();
+
+      console.log({ sceneProv });
```
Comment (failfmi) [inline src/modules/land-works/views/scene-expert-view/index.tsx:31]: I guess this is no longer needed

File: src/modules/land-works/views/single-scene-expert-view /index.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (failfmi) [inline src/modules/land-works/views/single-scene-expert-view /index.tsx:1]: There is an extra space in the folder `single-scene-expert-view ` name

**Review by failfmi — COMMENTED** [review]
Nice work!
There are only two browser console errors on the pages related to array keys.

File: src/modules/land-works/views/scene-expert-view/index.tsx (hunk 1 above)
Comment (zetske) [reply src/modules/land-works/views/scene-expert-view/index.tsx:31]: this PR is still WIP, need to finish up a few things

**Review by failfmi — APPROVED** [review]

**Pull request merged**